services/infinispan-device-registry/src/main/java/org/eclipse/hono/registry/infinispan/InfinispanRegistry.java

```diff
@@ -0,0 +1,42 @@
+/*******************************************************************************
+ * Copyright (c) 2019 Contributors to the Eclipse Foundation
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information regarding copyright ownership.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *******************************************************************************/
+
+package org.eclipse.hono.registry.infinispan;
+
+import org.eclipse.hono.deviceregistry.Application;
```

Comment:
IMHO we should not have a dependency on another registry implementation.

Reply:
See above ... But in the case of Application class I don't see any file-based registry specific code in it.